config/bindings/namespaced/kustomization.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+- ../../rbac
+- role_binding.yaml
+# Comment the following 3 lines if you want to disable
+# the auth proxy (https://github.com/brancz/kube-rbac-proxy)
+# which protects your /metrics endpoint.
+- auth_proxy_service.yaml
+- auth_proxy_role.yaml
+- auth_proxy_role_binding.yaml
+
+patchesJson6902:
+# Modify ClusterRole to Role
+- target:
```

Comment:
I am curious why we only have to modify one cluster role (`manager-role`), when I know there are four such "Cluster" resources: `manager-role`, `manager-rolebinding`, `proxy-role`, and `proxy-rolebinding`. Why are we taking different approach for `manager-role` vs. the others?

Reply:
The others, by default, are namespaced to the resources they are created. As long as we specify those resources to be a "Role" and not a "ClusterRole" (which we do in the `bindings` folder) then the bindings namespace properly. The manager role, though, is created by Kubebuilder, so we cannot make a copy of it - this would require engineers to modify the copy each time. Therefore we use a patch to force it to be a namespace Role.